README.md

```diff
@@ -10,52 +10,52 @@
 
 <h2 align="right">
         <img src="https://img.shields.io/badge/status-ongoing-blue?style=for-the-badge&logo=appveyor" alt="Status: Ongoing">   
-        <img src="https://img.shields.io/badge/sprint-1-blue?style=for-the-badge&logo=appveyor" alt="Second= sprint">
+        <img src="https://img.shields.io/badge/sprint-2-blue?style=for-the-badge&logo=appveyor" alt="Second= sprint">
 </h2>
             
 
 # Index 📎
 
 - [Index 📎](#index-)
-- [Sobre 📚](#sobre-)
+- [About 📚](#about-)
 - [Product Backlog 📍](#product-backlog-)
 - [User Backlog 📄](#user-backlog-)
-- [Calendário de Entregas 🗓](#calendário-de-entregas-)
-- [Projeto em Operação 📱](#projeto-em-operação-)
-- [Equipe 👩‍💻👨‍💻](#equipe-)
+- [Delivery Schedule 🗓](#delivery-schedule-)
+- [Project in Operation 📱](#project-in-operation-)
+- [Team 👩‍💻👨‍💻](#team-)
 
-# Sobre 📚
+# About 📚
 
-<p align="justify">O projeto constitui a adição de uma nova ferramenta para um aplicativo feito em outro projeto integrador para a Visiona. Nesta ferramenta era necessário ir a campo contar vagens e grãos para seu uso e isso exigia muito tempo do agricultor, surgiu a ideia de criar uma ferramenta com inteligência artificial para a contagem automática e visual desses grãos e esta é a ferramenta requisitada para nosso grupo.</p>
+<p align="justify">The project constitutes an addition of a new tool to an application made in another integrator project for Visiona. In this tool it was necessary to go to the field to count pods and grains for its use and this required time from the farmer, the idea arose of creating a tool with artificial intelligence for automatic and visual counting of these grains and this group is the required tool for ours.</p>
```

Comment:
"the idea arose of creating a tool with artificial intelligence for automatic and visual counting of these grains and this group is the required tool for ours." 

ficou com uma gramatica confusa, tente 

"the idea of creating a tool with artificial intelligence for automatic and visual counting of these grains was created and given to us to develop."

ou algo assim